constants.bzl

```
@@ -4,9 +4,9 @@ RELEASE_ENGINES_NPM = "^6.11.0 || ^7.5.6 || >=8.0.0"
 RELEASE_ENGINES_YARN = ">= 1.13.0"
 
 NG_PACKAGR_VERSION = "^20.0.0-next.0"
```

```suggestion
NG_PACKAGR_VERSION = "^20.1.0-next.0"
```

This was released last week on NPM.